vertx-web/src/test/java/io/vertx/ext/web/WebTestBase.java

```diff
@@ -37,13 +37,15 @@
   protected HttpServer server;
   protected HttpClient client;
   protected Router router;
+  protected int serverPort;
 
   @Override
   public void setUp() throws Exception {
     super.setUp();
+    serverPort = 8080;
```

Comment:
I think we need it because it's hard coded in  `testRequestBuffer` in `WebTestBase.java` and should be a variable to handle the test for both cases http1x and http2